landing/src/hooks/use-rewards.tsx

```
@@ -0,0 +1,54 @@
+        if (!season_pool_fee_recipient) {
+          console.error("Failed to get season pool fee recipient from config");
+          return;
+        }
+
+        // Convert address to hex string with 0x prefix
+        const recipientAddress = "0x" + season_pool_fee_recipient.toString(16);
+
+        // Get balance from contract
+        const balance = await getBalance(recipientAddress);
+
+        if (balance && balance[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)